src/shoobx/immutable/tests/test_revisioned.py

```diff
@@ -296,9 +296,16 @@ def test_rollbackToRevision_noActivation(self):
 
     def test_addRevision(self):
         rimm = revisioned.SimpleRevisionedImmutableManager()
+        rim = revisioned.RevisionedImmutable()
+
+        # can NOT add a transient object
+        with self.assertRaises(AssertionError):
+            rimm.addRevision(rim)
+
```

Comment:
Please make the error case its own test method called `test_addRevision_withTransientObject`.